kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h

```diff
@@ -1056,9 +1057,10 @@ class ResidualBasedEliminationBuilderAndSolver
         }
     }
 
-    
+
     //**************************************************************************
 	virtual void ConstructMatrixStructure(
+        typename TSchemeType::Pointer pScheme,
 		TSystemMatrixType& A,
 		ElementsContainerType& rElements,
 		ConditionsArrayType& rConditions,

```

Comment:
This function is not overriding anything (builder_and_solver.h does not have a ConstructMatrixStructure method)